README.md

```diff
@@ -218,6 +218,7 @@ Some [Groupware](#groupware) solutions also feature calendar/address book editin
   * [Actor](https://actor.im/) - Actor is a fast, open source messaging platform with rich mobile and web clients. ([Demo](https://app.actor.im), [Source Code](https://github.com/actorapp/actor-platform)) `AGPLv3` `Scala`
   * [Centrifugo](https://github.com/centrifugal/centrifugo) - Language-agnostic real-time messaging (Websocket or SockJS) server. ([Demo](https://github.com/centrifugal/centrifugo#demo)) `MIT` `Go`
   * [Cherry](https://github.com/rafael-santiago/cherry) - A tiny webchat server. `GPLv2` `Go`
+  * [DeadSimpleScreenSharing](https://github.com/mlakkadshaw/DeadSimpleScreeSharing)  - Open Source multi-user screen sharing solution right from the browsers. `HTML5/NODE.JS`
```

Comment:
@mlakkadshaw Thanks, please use `Nodejs` as the language (just with capital N). HTML is not needed since it's the client side language, and well all web applications use html anyway.

The license tag is also missing (MIT)